src/main/java/com/facebook/openwifirrm/modules/ApiServer.java

```diff
@@ -110,9 +110,15 @@
 	scheme = "bearer"
 )
 public class ApiServer implements Runnable {
+	private static final String SPARK_EMBEDDED_SERVER_IDENTIFIER =
+		ApiServer.class.getName();
+
 	private static final Logger logger =
 		LoggerFactory.getLogger(ApiServer.class);
 
+	/** The Spark service instance */
```

Comment:
can you add a note why this is needed (for future reference)? since it's unusual to see in Spark applications